dynawo/3rdParty/libiidm/extensions/voltageRegulation/xml/src/VoltageRegulationHandler.cpp

```diff
@@ -37,8 +37,8 @@ namespace voltageregulation {
 namespace xml {
 
 const std::string & VoltageRegulationHandler::xsd_path() {
-  static std::string XSD_PATH = IIDM_EXT_VOLTAGEREGULATION_XML_XSD_PATH + std::string("voltageRegulation.xsd");
-  return XSD_PATH;
+  static std::string xsdPath = getMandatoryEnvVar("IIDM_XML_XSD_PATH") + std::string("voltageRegulation.xsd");
```

Comment:
why not const as above?

Reply:
i will harmonize it on the master but it has no impact in v1.2.2 has we don't even check xsd at runtime in real usecases

Reply:
ok, thx!